src/protocols/rdp/Makefile.am

```diff
@@ -269,6 +269,7 @@ rdp_keymaps =                   \
     $(srcdir)/keymaps/en_us_qwerty.keymap \
     $(srcdir)/keymaps/es_es_qwerty.keymap \
     $(srcdir)/keymaps/fr_fr_azerty.keymap \
+    $(srcdir)/keymaps/fr_be_azerty.keymap \
```

Comment:
Nitpick - this should probably be *above* the `fr_fr` keymap, as should the `fr_ch` one.

Reply:
They are now alphabetically sorted.